SampleGallery/Samples/SDK 16299/LinearGradients/LinearGradients.xaml.cs

```diff
@@ -0,0 +1,257 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, 
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. 
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, 
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, 
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH 
+// THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+//*********************************************************
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Hosting;
+using Windows.UI;
+using System.Numerics;
+
+namespace CompositionSampleGallery
+{
+    public sealed partial class LinearGradients : SamplePage
+    {
+        public LinearGradients()
+        {
+            this.InitializeComponent();
+        }
+
+        public static string StaticSampleName => "Linear Gradients";
+        public override string SampleName => StaticSampleName;
+        public static string StaticSampleDescription => "Simple visual effect showing linear gradient animations.The first and last words will flip the colors from left to right.Clicking on any of the words will change the color scheme from warm to cool and back when clicked again." + Environment.NewLine + "This sample is to show how linear gradients can highlight content in colorful ways.";
+        public override string SampleDescription => StaticSampleDescription;
+
+        private Compositor _compositor;
+
+        private SpriteVisual _vis1;
+        private SpriteVisual _vis2;
+        private SpriteVisual _vis3;
+        private SpriteVisual _vis4;
+
+        private CompositionLinearGradientBrush _brush1;
+        private CompositionColorGradientStop b1GradientStop1;
+        private CompositionColorGradientStop b1GradientStop2;
+
+
+        private Vector3KeyFrameAnimation _scaleAnim;
+        private ScalarKeyFrameAnimation _offsetAnim;
+
+        private ColorKeyFrameAnimation _colorAnimBrush1Stop1;
```

Comment:
I'm confused by these names.  What is the difference between the _colorAnimBrushxxxx pair and the _changexxxx pair?